assign_rights/test_helpers.py

```diff
@@ -28,31 +34,103 @@ def add_groupings(count=5):
                 grouping.rights_shells.add(random.choice(RightsShell.objects.all()))
 
 
+def set_dates():
+    """Returns start and end dates in a way consistent with what is allowed in forms"""
+    start_date = None
+    end_date = None
+    start_date_period = None
+    end_date_period = None
+    end_date_open = False
+    choice = random.randrange(1, 5)
+    if choice == 1:
+        start_date = random_date(100, 25)
+        end_date = random_date(24, -50)
+    if choice == 2:
+        start_date = random_date(100, 25)
+        end_date_period = random.randint(20, 75)
+    if choice == 3:
+        start_date_period = random.randint(0, 10)
+        end_date_period = random.randint(20, 75)
+    else:
+        start_date_period = random.randint(0, 10)
+        end_date_open = True
+    return start_date, end_date, start_date_period, end_date_period, end_date_open
+
+
 def add_rights_shells(count=15):
-    for x in range(count):
+    def copyright_shell():
+        date_info = set_dates()
```

Comment:
Wondering if it makes sense to call this at line 121 and then pass the result as an argument to these various `*_shell` methods?